plugins/apps/subcommands/report

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_AVAILABLE_PATH/apps/functions"
+
+report_single_app() {
+  local APP="$1"; local APP_DIR="$DOKKU_ROOT/$APP"; local INFO_FLAG="$2"
+  verify_app_name "$APP"
+  local DOKKU_APP_CIDS=$(get_app_container_ids "$APP")
+  if [[ -n $DOKKU_APP_CIDS ]]; then
+    local cid
+    for CID in $DOKKU_APP_CIDS; do
+      local APP_CONTAINER_STATUS
+      local DOKKU_APP_CID
+      DOKKU_APP_CID=$CID
+      APP_CONTAINER_STATUS=$(docker inspect -f '{{.State.Status}}' "$CID")
+    done
+  fi
+  local flag_map=(
+    "--app-dir: $APP_DIR"
+    "--git-sha: $(GIT_DIR="$APP_DIR" git rev-parse --short HEAD 2> /dev/null || false)"
+    "--app-cid: ${DOKKU_APP_CID:0:12}"
+    "--status: $APP_CONTAINER_STATUS"
+  )
+
+  if [[ -z "$INFO_FLAG" ]]; then
+    dokku_log_info2 "$APP"
+    if (is_deployed "$APP"); then
+      for flag in "${flag_map[@]}"; do
+        key="$(echo "${flag#--}" | cut -f1 -d' ' | tr - ' ')"
+        dokku_log_verbose "$(printf "%-20s %-25s" "${key^}" "${flag#*: }")"
+      done
+    else
+      echo "not deployed"
```

Comment:
Should use our error helpers. If there isnt one that doesnt output to stderr without exiting, create one :)